### PR TITLE
Changes array to normalize to empty array.

### PR DESCRIPTION
**Pull request by justinlittman**
**NOTE:  Changes to openapi.yml require updating openapi.yml for sdr-api and dor-services-app and generating models - see README.**

## Why was this change made?
So that logically equivalent Cocina models normalize to equivalent hashes.

## How was this change tested?
Unit

### CODE REVIEW

File: spec/cocina/models/administrative_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    end
+
+    let(:administrative2) do
+      described_class.new({
+                            hasAdminPolicy: 'druid:bc123df4567',
+                            releaseTags: []
+                          })
+    end
+
+    it 'normalizes array' do
```
Comment (justinlittman) [inline spec/cocina/models/administrative_spec.rb:20]: This shows why the change is desirable.

**Review by jcoyne — APPROVED** [review]
